### DISCUSSION TITLE
Add support for sending logs to a text file

### DISCUSSION THREAD

**Discussion by meareal** [Azure/data-api-builder#1812] [Ideas]
In `development` mode, we get logs in the console. Ideally we would be able to log every request and response to a text file we specify in dab-config.json file.

In the past, I have added a middleware component to our custom Web APIs to log every request and every response in the `InvokeAsync` method.

**Comment by seantleonard** [comment]
Thank you for your suggestion, @meareal. We are now tracking this ask in #1818

**Comment by JerryNixon** [comment]
https://github.com/Azure/data-api-builder/issues/2371
